index.html

```diff
@@ -22,7 +22,7 @@ <h1 class="text-center">What is Rook?</h1>
         </div>
         <div class="col-6_md-12">
             <h2>A Storage Orchestrator for Kubernetes</h2>
-            <p>Rook turns distributed storage systems into self-managing, self-scaling, self-healing storage services. It automates the tasks of a storage administrator: deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management.<br /><br />Rook uses the power of the Kubernetes platform to deliver its services: cloud-native container management, scheduling, and orchestration.</p>
+            <p>Rook turns distributed storage systems into self-managing, self-scaling, self-healing storage services. It automates the tasks of a storage administrator: deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management.<br /><br />Rook uses the power of the Kubernetes platform to deliver its services via a Kubernetes Operator for each storage provider.</p>
```

Comment:
this wording change doesn't seem really needed to me, the current text is referring to what Kubernetes brings to the table, right? "cloud-native container management, scheduling, and orchestration"

Reply:
These three terms: "cloud-native container management", "scheduling", and "orchestration" represent the same thing to me and they seem unnecessary. They're on rook.io to learn about Rook so I was trying to emphasize what Rook provides. 
I'm thinking the title of this section should be "Storage Operators for Kubernetes". We don't have a single "storage orchestrator for Kubernetes". I think this title change will go well with the new text as well. Let me know what you think.

Reply:
That seems reasonable!